PROV/prov-cco-directmappings.ttl

```diff
@@ -19,46 +19,68 @@
+prov:wasInformedBy        rdfs:subPropertyOf cco:caused_by ;
+                          rdfs:comment "If the Activity A1 was informed by the Activity A2, then A1 is caused by A2"@en .
+
+prov:invalidated          rdfs:subPropertyOf cco:affects ;
+                          rdfs:comment "An Activity that invalidated an Entity also affects that Entity."@en .
+prov:used                 rdfs:subPropertyOf cco:affects ;
+                          rdfs:comment "An Activity that used an Entity affects the Entity."@en .
+prov:generated            rdfs:subPropertyOf cco:has_output ;
+                          rdfs:comment "An Activity that generated an Entity has_output that Entity"@en .
 
 
 # "The _optional_ Plan adopted by an Agent in Association with some Activity. Plan specifications are out of the scope of this specification."
-prov:hadPlan                  rdfs:subPropertyOf cco:has_input . # an Influence has_input some Plan
+prov:hadPlan                  rdfs:subPropertyOf cco:has_input ; # an Influence has_input some Plan
+                              rdfs:comment "If an Association had plan some Plan, then it had that Plan as an input"@en .
 
 # "If this Activity prov:wasAssociatedWith Agent :ag, then it can qualify the Association using prov:qualifiedAssociation [ a prov:Association;  prov:agent :ag; :foo :bar ]."
-prov:qualifiedAssociation     rdfs:subPropertyOf  cco:has_process_part .
-prov:qualifiedAttribution     rdfs:subPropertyOf cco:is_affected_by .
-prov:qualifiedRevision        rdfs:subPropertyOf cco:is_affected_by .
+prov:qualifiedAssociation     rdfs:subPropertyOf  cco:has_process_part ;
+                              rdfs:comment "An Activity that is associated with an Agent has that Association has a process part"@en .
+prov:qualifiedAttribution     rdfs:subPropertyOf cco:is_affected_by ;
+                              rdfs:comment "An Entity that is attributed to something is affected by that Attribution"@en .
+prov:qualifiedRevision        rdfs:subPropertyOf cco:is_affected_by ;
+                              rdfs:comment "If a Revision is the qualified revision of an Entity, then the Entity is affected by the Revision.  "@en .
 
-prov:qualifiedDelegation      rdfs:subPropertyOf cco:agent_in .
-prov:qualifiedDerivation      rdfs:subPropertyOf cco:is_output_of .
-prov:qualifiedPrimarySource   rdfs:subPropertyOf cco:is_output_of .
+prov:qualifiedDelegation      rdfs:subPropertyOf cco:agent_in ;
+                              rdfs:comment "If a Delegation is the qualified delegation of an Agent, then the Agent is agent in (namely causally active in) the Delegation. "@en .
+prov:qualifiedDerivation      rdfs:subPropertyOf cco:is_output_of ;
+                              rdfs:comment "If a Derivation is the qualified derivation of an Entity, then the Entity is output of the Derivation."@en .
+prov:qualifiedPrimarySource   rdfs:subPropertyOf cco:is_output_of ;
+                              rdfs:comment ""@en .
```

Comment:
A Entity that has a Primary Source as its qualified primary source is output of that Primary Source.